src/gui/app.cpp

```diff
@@ -104,17 +246,14 @@ void App::showUi() noexcept
 		win->show();
 	}
 #else
-	NeovimQt::MainWindow *win = new NeovimQt::MainWindow(m_connector.get());
+	// FIXME: On macOS, Cmd+q kills the first window that was created.
```

Comment:
No FIXME or un-tracked TODO comments please :)

Two options here:
1. Fix the underlying issue now (preferred).
2. File an issue and reference the issue number: `// TODO Issue#??: On macOS, CMD+q ...`

Is this a bad enough issue that we should fix it now, before check-in?

Reply:
My bad. I actually fixed that but forgot to remove that FIXME. I'll remove it.

Reply:
Excellent! :+1: